src/graphicalClient/soccerview.cpp

```diff
@@ -170,6 +170,8 @@ void GLSoccerView::resizeEvent(QResizeEvent* event)
 {
   QGLWidget::resizeEvent(event);
   redraw();
+
+  resetView(); 
```

Comment:
What is that supposed to do? As far as I can tell this has nothing to do with fixing memory leaks but changes the behavior when resizing the window. If i remember correctly, the old behavior was that resizing the window doesn't change the size of the rendered field. The change would always reset the field size to its default when the window is resized.
Why still redraw the widget before resetView is called? The latter calls postRedraw() which queues a call to redraw in the Qt event loop.